hack/tools/mdbook/releaselink/releaselink.go

```diff
@@ -58,36 +56,22 @@ func (l ReleaseLink) Process(input *plugin.Input) error {
 		versionRange := semver.MustParseRange(tags.Get("version"))
 		includePrereleases := tags.Get("prereleases") == "true"
 
-		repo, err := vcs.RepoRootForImportPath(gomodule, false)
+		scheme, host, err := goproxy.GetSchemeAndHost(os.Getenv("GOPROXY"))
```

Comment:
Since now we are reading from `GOPROXY` env variable we should account for `scheme` and `host` being empty strings. 

If `GOPROXY` envvar is set to `direct` or `off` we will get an empty `scheme` and `host` and the code below will fail.

Reply:
Added returning an error when one of scheme or host is empty to fail.